deployment/sriov-network-operator/README.md

```diff
@@ -42,6 +42,9 @@ For additional information and methods for installing Helm, refer to the officia
 # Install Operator
 $ helm install -n sriov-network-operator --create-namespace --wait sriov-network-operator ./
 
+# Add PSA label for the sriov network operator namespace
+$ kubectl label ns sriov-network-operator pod-security.kubernetes.io/enforce=privileged
```

Comment:
Do we still need to do this step? The deploy/namespace.yaml should take care of it right?

Reply:
yep because the helm deployment doesn't use the deploy folder

Reply:
Got it thanks!

Reply:
by default if PSA is enabled AFAIU it will do nothing unless namespace has been labeled.

is my understanding correct ?

if yes, then we should have a section that says, in case PSA is used, privileged policy should be set in enforce label.
an not always add the label.